packages/calcite-components/src/components/select/select.e2e.ts

```diff
@@ -422,6 +422,48 @@ describe("calcite-select", () => {
     await assertSelectedOption(page, await page.find("calcite-option[value='']"));
   });
 
+  it("Handles type error when disconnected while loading", async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(`
```

Comment:
For consistency, can you use the `newProgrammaticE2EPage` + E2E/Puppeteer APIs to drive set up and drive this test? [Here's](https://github.com/Esri/calcite-design-system/blob/bce91dffedee8c581842867f54e23e5847e265a7/packages/calcite-components/src/components/list/list.e2e.ts#L933-L979) an example.